src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:nth-child(-n + 4) {
+      background-image: url('../images/star-active.svg');
```

Comment:
The URL in the background-image property should be consistent with the one used in line 7. Consider using double quotes for both or single quotes for both to maintain consistency.